src/zugferd211/Model/TradeAllowanceCharge.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Easybill\ZUGFeRD211\Model;
+
+use JMS\Serializer\Annotation as JMS;
+
+/**
+ * Class AllowanceCharge.
+ */
+class TradeAllowanceCharge
+{
+    /**
+     * @JMS\Type("Easybill\ZUGFeRD211\Model\Indicator")
+     * @JMS\XmlElement(cdata=false, namespace="urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:100")
+     * @JMS\SerializedName("ChargeIndicator")
+     */
+    public Indicator $indicator;
+
+    /**
+     * @JMS\Type("Easybill\ZUGFeRD211\Model\Amount")
+     * @JMS\XmlElement(cdata=false, namespace="urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:100")
+     * @JMS\SerializedName("BasisAmount")
+     */
+    public Amount $basisAmount;
+
+    /**
+     * @JMS\Type("Easybill\ZUGFeRD211\Model\Amount")
+     * @JMS\XmlElement(cdata=false, namespace="urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:100")
+     * @JMS\SerializedName("ActualAmount")
+     */
+    public Amount $actualAmount;
+
+    /**
+     * @JMS\Type("string")
+     * @JMS\XmlElement(cdata = false, namespace = "urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:100")
+     * @JMS\SerializedName("Reason")
+     */
+    public string $reason;
+
+    /**
+     * @var TradeTax[]
+     * @JMS\Type("array<Easybill\ZUGFeRD211\Model\TradeTax>")
+     * @JMS\XmlList(inline = true, entry = "CategoryTradeTax", namespace="urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:100")
+     */
+    public array $tradeTax;
```

Comment:
```suggestion
    public array $tradeTax = [];
```

Reply:
The $actualAmount is not required either, is it?

Reply:
$actualAmount is required.
<img width="736" alt="Bildschirmfoto 2020-11-16 um 09 59 38" src="https://user-images.githubusercontent.com/318564/99232678-83b24780-27f2-11eb-8583-8622d3810ecd.png">